gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/DagProcUtils.java

```diff
@@ -199,6 +202,21 @@ private static void sendCancellationEvent(Dag.DagNode<JobExecutionPlan> dagNodeT
     jobExecutionPlan.setExecutionStatus(CANCELLED);
   }
 
+  public static void setAndEmitFlowEvent(EventSubmitter eventSubmitter, Dag<JobExecutionPlan> dag, String flowEvent) {
+    if (!dag.isEmpty()) {
+      // Every dag node will contain the same flow metadata
+      Config config = DagManagerUtils.getDagJobConfig(dag);
+      Map<String, String> flowMetadata = TimingEventUtils.getFlowMetadata(config);
+      dag.setFlowEvent(flowEvent);
```

Comment:
do we set this, but don't actually persist it in the store?

Reply:
yes, thats a todo for blr team.
flow event is not persisted in mysql, it never was.

Reply:
i am still setting it here, so that future team can decide what to do with this field, for now marked it as @Deprecated